src/main/java/at/overflow/bukkit/matrixbridge/BridgePropertyReader.java

```diff
@@ -66,7 +68,12 @@ public String getPassword() {
     }
 
     public String getRoom() {
-      String tmp = properties.getProperty(MATRIX_ROOMID);
+      String tmp = properties.getProperty(MATRIX_ROOMID_KEY);
       return tmp;
     }
+
+  public String getAdmin() {
+      String tmp = properties.getProperty(MATRIX_ADMINID_KEY);
+      return tmp;
```

Comment:
I think I started doing this with `getPassword` and `getMinecraftServerName` but it's probably cleaner to remove the temporary variable.

Reply:
Will fix it in this PR, too.